cli/tests/test_git_private_commits.rs

```diff
@@ -88,9 +88,11 @@ fn test_git_private_commits_block_pushing() {
     // Will not push when a pushed commit is contained in git.private-commits
     test_env.add_config(r#"git.private-commits = "description(glob:'private*')""#);
     let stderr = test_env.jj_cmd_failure(&workspace_root, &["git", "push", "--all"]);
-    insta::assert_snapshot!(stderr, @r###"
```

Comment:
ℹ️  The triple octothorpes `###` were removed automatically during my run of `cargo insta test --review --workspace --test-runner nextest`. I'm using `cargo-insta 1.41.1` and `cargo-nextest 0.9.82`. Please let me know if I should restore that part of the diff, or adjust my command invocation in some way.

Reply:
It's new `cargo insta` behavior. No need to preserve the old style.